service-map/src/main/java/fi/nls/oskari/map/layer/OskariLayerServiceMybatisImpl.java

```diff
@@ -302,6 +302,18 @@ public OskariLayer find(int id) {
         }
         return Collections.emptyList();
     }
+    public List<String> findAllLayerNames () {
```

Comment:
Should this be "for url"?

Reply:
When we have something like "buildings" on two different services this doesn't make much sense.